benchmarks/dynamo/pr_time_benchmarks/expected_results.csv

```diff
@@ -58,7 +58,7 @@ aotdispatcher_partitioner_cpu,compile_time_instruction_count,9158000000,0.015
 
 
 
-aotdispatcher_training_nosubclass_cpu,compile_time_instruction_count,3863000000,0.015
+aotdispatcher_training_nosubclass_cpu,compile_time_instruction_count,3932045322,0.015
```

Comment:
I see we're regressing a compile-time AOT benchmark by 1.7%. Not... too bad? But if there's anything dumb we're doing it's probably worth fixing now

Reply:
it feels like we should really have any regression at all, since any extra work we're doing at compile time in this PR should only really apply to nested ints, right?

Reply:
lol now its actually complaining about this PR beating the benchmark by 1.7% !
I think it was a preexisting issue/noise somewhere

Reply:
cc @laithsakka (do we generally expect this benchmark to be noisy? I would have thought no since it's deterministic)